src/Store/DoctrineDbalStore.php

```diff
@@ -65,6 +73,9 @@ public function __construct(
         $this->config = array_merge([
             'table_name' => 'eventstore',
             'aggregate_id_type' => 'uuid',
+            'locking' => true,
+            'lock_id' => self::DEFAULT_LOCK_ID,
+            'lock_timeout' => -1,
```

Comment:
Maybe 
```php
[
  'lock' => [
      'enabled' => true,
      'id' => '',
      'timeout' => -1
   ]
]
```

Reply:
I would like to leave the config here flat, anything else just makes it much more complicated.

Reply:
Im not sure if it makes it really complicated. I thought more about clustering it since the 3 option are tied together and makes it clearer from a user pov. I'm fine with both.